src/components/static-map.js

```diff
@@ -204,14 +251,18 @@ export default class StaticMap extends PureComponent {
     return null;
   }
 
-  _renderOverlays(dimensions) {
+  _renderOverlays(dimensions : {
+    width?: number,
+    height?: number
+  }) {
     const {
-      width = this.props.width,
-      height = this.props.height
+      width = Number(this.props.width),
```

Comment:
These will convert to `NaN` if width or height is not supplied. Looks fine in this context but something to be aware of if we do a `typeof width === 'number'` check later on

Reply:
The callback will only be missing width/height if they are disabled on AutoSizer: https://github.com/uber/react-map-gl/blob/master/src/components/static-map.js#L306

Overall I struggled a lot with flow when using type overrides. e.g. StaticMap and InteractiveMap support `width: number | string` but Mapbox and MapController only accept `width: number`. Flow does not like it when I pass props from one to another using `Object.assign`.